### ISSUE TITLE
Build fails on systems with case insensitive file system

### ISSUE THREAD

**Issue by 4ever2** [AU-COBRA/ConCert#84]
Running `make` fails on systems with case insensitive file system due to uncapitalization step done in `extraction/process_extraction.sh`

**Comment by jakobbotsch** [comment]
Are you running .sh files through Cygwin or how? If this is WSL, can you just store the ConCert folder on the Linux partition instead of the Windows partition as a workaround?

**Comment by 4ever2** [comment]
I am running through WSL and it is possible to toggle between case sensitive and insensitive file system per directory, so I already have a workaround.

**Comment by spitters** [comment]
@jakobbotsch Should we report this upstream in metacoq?

**Comment by jakobbotsch** [comment]
Yes that would make sense, I assume the same problem is there when installing MetaCoq?

**Comment by spitters** [comment]
Yes, metacoq has the same problem. I did not want to report it there myself, as I have no way to test.
@jakobbotsch could you tell them?

**Comment by spitters** [comment]
@jakobbotsch did you ever report this, or shall I do it?

**Comment by jakobbotsch** [comment]
@spitters I didn't get around to it, feel free to report it
